docs/GlutenUsage.md

```diff
@@ -1,20 +1,27 @@
 ### Build parameters
 
-Based on the different environment, there are some parameters can be set via -D with mvn.
+There are some parameters can be set via -- with one_step_veloxbackend.sh.
 
-| Parameters                                            | Description                                                                                                                                                                         | Default Value                                       |
-|-------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------|
-| backends-velox                                        | Add -Pbackends-velox in maven command to compile the JVM part of Velox backend.                                                                                                     | false                                              |
-| backends-gazelle                                        | Add -Pbackends-gazelle in maven command to compile the JVM part of Gazelle backend.                                                                                                     | false                                              |
-| backends-clickhouse                                   | Add -Pbackends-clickhouse in maven command to compile the JVM part of ClickHouse backend.                                                                                           | false                                              |
+| Parameters | Description | Default value |
+| ---------- | ----------- | ------------- |
+| build_type | Gluten build type(for arrow/velox/cpp)  | release |
+| build_test | build test code in cpp folder and arrow | OFF |
+| build_benchmarks | build benchmark code in cpp folder and arrow| OFF |
+| build_jemalloc   | build with jemalloc | ON |
+| enable_hbm | enable HBM allocator      | OFF|
+| build_protobuf | build protobuf lib    | ON|
+| enable_s3   | build with s3 lib        | OFF|
+| enable_hdfs | build with hdfs lib      | OFF|
+| build_folly | build folly lib    | ON|
+| build_arrow_from_source   | pull the source code and build arrow lib| ON|
+| build_velox_from_source   | pull the source code and build velox lib| ON|
 
 There are some parameters can be set via -- with build_arrow_for_gazelle.sh.
 
 | Parameters | Description | Default value |
 | ---------- | ----------- | ------------- |
 | arrow_home | Arrow build path            | GLUTEN_DIR/ep/build-arrow/build|
-| build_type | ARROW build type            | release|
-| enable_ep_cache |  Use cache lib when arrow is not change |  OFF   |      
```

Comment:
Is there a way to enable ep cache from one step script?

Reply:
Sorry, I forgot. will add enable_ep_cache in one_step_veloxbackend.sh.